lib/tlitest/test_tlog_rec.py

```diff
@@ -46,7 +46,7 @@ def test_record_command_to_journal(self):
         shell = ssh_pexpect(self.user1, 'Secret123', 'localhost')
         reccmd = 'test_record_to_journal'
         shell.sendline('tlog-rec -w journal echo {}'.format(reccmd))
-        find_journal_entry_with_match(reccmd, match_filter, 10)
```

Comment:
Should there be a call to `check_journal` here? I believe that the higher level function which calls `journal_find_last`

Reply:
yes, correcting now.